modules/36_buildings/simple/datainput.gms

```diff
@@ -57,6 +57,17 @@ $offdelim
 ;
 p36_uedemand_build(ttot,regi,in) = f36_uedemand_build(ttot,regi,"%cm_demScen%","%cm_rcp_scen_build%",in);
 
+*** Scale UE demand and floor space in the building sector
+$ifthen.scaleDemand not "%cm_scaleDemand%" == "off"
+  loop((t,t3,regi) $ pm_scaleDemand(t,t3,regi),
+      loop(t2,
+*FL*      rescaled demand               = normal demand                 * [ scaling factor            + (1-scaling factor)            * remaining phase-in, between zero and one       ]
+          p36_uedemand_build(t,regi,in) = p36_uedemand_build(t,regi,in) * ( pm_scaleDemand(t,t3,regi) + (1-pm_scaleDemand(t,t3,regi)) * min(1, max(0, t3.val-t2.val) / (t3.val-t.val)) );
+          p36_floorspace(t,regi)        = p36_floorspace(t,regi)        * ( pm_scaleDemand(t,t3,regi) + (1-pm_scaleDemand(t,t3,regi)) * min(1, max(0, t3.val-t2.val) / (t3.val-t.val)) );
```

Comment:
Applying the same reduction factor to floor space is a strong assumption: You assume that the entire reduction comes from floor space reduction and not additional building shell efficiency, behaviour or so. But I guess it is in line with the other sectors. I might be wrong but I think that if you rescale all industry quantities by the same factor, this also represents a reduction of material demand and not any material/energy efficiency, etc.

Reply:
Just for the record:
Decision was made to use exponent 0.3 for floor space, assuming that the rest of the demand variation is due to other measures.